src/Tizen.NUI/src/public/Images/NativeImageQueue.cs

```diff
@@ -0,0 +1,109 @@
+
+namespace Tizen.NUI
+{
+    using global::System;
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NativeImageQueue : NativeImageInterface
+    {
+        private IntPtr handle;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NativeImageQueue(uint width, uint height, NativeImageSource.ColorDepth depth) : this(Interop.NativeImageQueue.NewHandle(width, height, (int)depth), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal NativeImageQueue(IntPtr cPtr, bool cMemoryOwn) : base(Interop.NativeImageQueue.Get(cPtr), cMemoryOwn)
+        {
+            handle = cPtr;
+        }
+
+        /// <summary>
+        /// Generate Url from native image queue.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageUrl GenerateUrl()
+        {
+            ImageUrl ret = new ImageUrl(Interop.NativeImageSource.GenerateUrl(this.SwigCPtr.Handle), true);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
+            return ret;
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.NativeImageQueue.Delete(handle);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal static global::System.Runtime.InteropServices.HandleRef getCPtr(NativeImageQueue obj)
+        {
+            return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.SwigCPtr;
+        }
+
+        /// <summary>
+        /// Checks if the buffer can be got from the queue.
+        /// </summary>
+        /// <example>
+        /// <code>
+        ///
+        /// NativeImageQueue queue = new NativeImageQueue(width,height,ColorDepth.Default);
+        /// if(queue.CanDequeueBuffer())
+        /// {
+        ///   var buffer = queue.DequeueBuffer(ref _width,ref _height,_stride);
+        ///   /* use buffer */
+        ///   queue.EnqueueBuffer(buffer);
+        /// }
+        ///
+        /// </code>
+        /// </example>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool CanDequeueBuffer()
+        {
+            bool ret = Interop.NativeImageQueue.CanDequeueBuffer(this.SwigCPtr.Handle);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// Dequeue buffer from the queue.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IntPtr DequeueBuffer(ref int width, ref int height, ref int stride)
```

Comment:
And here for parameters. Please add `<param>` descriptions.

Reply:
Thank you, I updated.